onnx/defs/tensor/defs.cc

```diff
@@ -7,19 +7,21 @@
 #include <cmath>
 
 namespace ONNX_NAMESPACE {
-static const char* Cast_ver6_doc = R"DOC(
+static const char* Cast_ver9_doc = R"DOC(
 The operator casts the elements of a given input tensor to a data type
 specified by the 'to' argument and returns an output tensor of the same size in
 the converted type. The 'to' argument must be one of the data types specified
-in the 'DataType' enum field in the TensorProto message.
-NOTE: Casting to and from strings is not supported yet.
+in the 'DataType' enum field in the TensorProto message. Litrals string 'NaN' 
```

Comment:
You should explicitly say when converting `String` tensor to numeric type of tensors. Also you should say which numeric representation supported.